packages/cf-style-const/src/variables.js

```diff
@@ -33,7 +33,8 @@ export default {
     grass: '#9BCA3E',
     marine: '#2F7BBF',
     tangerine: '#FF7900',
-    apple: '#BD2527'
+    apple: '#BD2527',
+    dessy: '#F7F7F7'
```

Comment:
Mmm.. `colorGrayLighter`? 😄 I'm really not sure what to call it but just maybe connect it more to what the value is exactly.

Otherwise I'd be happy to just have a very simple color pallet of our most frequently used colors and just hardcode the one-offs (this feels like a one off) in the themes of the components using it. If it's not gonna be used as the light gray color everywhere then it probably should be localized to where it's used.